include/votca/xtp/qmatom.h

```diff
@@ -67,6 +67,8 @@ class QMAtom {
 
   Index getNuccharge() const { return _nuccharge - _ecpcharge; }
 
+  Index getTotalNuccharge() const { return _nuccharge; }
```

Comment:
I am not super happy with the name, Is this still needed?

Reply:
Yes, but the name could be changed, getTrueNuccharge, getOriginalNuccharge, something like that?

Reply:
Index getElementNumber() ?

Reply:
nice